.release/template.json

```diff
@@ -0,0 +1,58 @@
+{
+
+  "variables": {
+    "aws_access_key_id": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_access_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
+    "wallaroo_version": "{{ env `WALLAROO_VERSION`}}"
+  },
+
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "access_key": "{{user `aws_access_key_id`}}",
+      "secret_key": "{{user `aws_secret_access_key`}}",
+      "instance_type": "m5.large",
+      "ssh_username": "ubuntu",
+      "ami_name": "wallaroo-{{ user `wallaroo_version`}}-{{timestamp}}",
+      "ami_regions": "us-west-2",
```

Comment:
Any reason why we only support `us-west-2` region for the AMI?

Reply:
The only reason is that I didn't want to dump a bunch of AMIs into our AWS account before we've converged on the shape of the end product. Less cleanup that way if we want to fundamentally change our approach before this goes live 'for real'.

Reply:
@dipinhora regarding the custom Vargrant boxes: Yes! I think using the same tool for as much as possible is a worthwhile goal, because the pipeline benefits from cross-testing on different targets.

Reply:
Fair enough re: regions and waiting to add them.